docs: update ts configure module structure (#10270)
dagger/dagger: docs/current_docs/api/module-structure.mdx

The TypeScript SDK is installed automatically, including dependencies, with a version that's tied to the currently running Dagger Engine container:

By default, the SDK is installed as a bundled local dependency, so all SDK related dependencies are already pre-bundled in the module's code

> and it doesn't add extra dependencie to the module's `package.json` file.

Reviewer: ```suggestion
and extra dependencies are not added to the module's `package.json` file.
```

```shell
# executed by the runtime container
yarn install
```

The SDK files are mounted under `/sdk` in the Dagger Engine runtime container.

> This is why the initial `package.json` doesn't include any dependencies except TypesScript.

Reviewer: ```suggestion
This is why the initial `package.json` doesn't include any dependencies except TypeScript.
```
